src/schema/notifications.ts

```diff
@@ -1,4 +1,5 @@
 import gql from 'graphql-tag';
+import assign from 'lodash/assign';
```

Comment:
lodash-es

Reply:
In api-js-client, we don't have lodash-es module. Should I install it?